lib/generators/component/templates/view.html.erb.erb

```diff
@@ -1,3 +1,7 @@
-<div class="<%= component_class_name %>">
-  <%= component_name %>
+<div data-controller="<%= component_class_name %>" class="<%= component_class_name %>">
```

Comment:
Shouldn't the `data-controller` be added only when stimulus is enabled?

Reply:
yes